.github/workflows/pr-make.yml

```diff
@@ -11,7 +11,7 @@ jobs:
           go-version: 1.15
       - name: Setup dependencies
         run: |
-          sudo apt-get install -y libgpgme-dev libdevmapper-dev btrfs-progs libbtrfs-dev
+          sudo apt-get update && sudo apt-get install -y libgpgme-dev libdevmapper-dev btrfs-progs libbtrfs-dev
```

Comment:
is this intended?

Reply:
yeah, that was part of the fix to get the CI going as suggested by GH folks